app/Http/Controllers/Account/AcceptanceController.php

```diff
@@ -119,6 +124,21 @@ public function store(Request $request, $id)
             $return_msg = trans('admin/users/message.declined');
         }
 
+        $item = $acceptance->checkoutable_type::find($acceptance->checkoutable_id);
+
+        $data = [
+            'item' => $item,
+            'eula' => $item->getEula(),
+            'signature' => storage_path().'/private_uploads/signatures/'.$sig_filename,
+            'logo' => public_path().'/uploads/snipe-logo.png',
+        ];
+
```

Comment:
Based on the $item we just pulled, let's get the actual EULA that's associated with it. 

TODO: Actually pull the settings logo, instead of hard-coding it like a punk.